src/GitHub.App/Services/ModelService.cs

```diff
@@ -366,6 +375,158 @@ IObservable<IReadOnlyList<IRemoteRepositoryModel>> GetOrganizationRepositories(s
                     });
         }
 
+#pragma warning disable CS0618 // DatabaseId is marked obsolete by GraphQL but we need it
```

Comment:
🚲 You could create a `GetDatabaseId` method surrounded by `#pragma warning disable CS0618` to avoid turning off obsolete warnings for a large block.

Reply:
That won't work here as these expressions are converted at runtime to GraphQL queries, and the expression rewriter can't introspect such a method, so building the query will fail.